htdocs/core/lib/modulebuilder.lib.php

```diff
@@ -416,3 +416,34 @@ function rebuildObjectSql($destdir, $module, $objectname, $newmask, $readdir = '
 
 	return $error ? -1 : 1;
 }
+
+/**
+ * Get list of existing objects from directory
+ * @param	string      $destdir		Directory
+ * @return Array|int                    <=0 if KO, array if OK
+ */
+function dolGetListOfObjectclasses($destdir)
```

Comment:
You might want to change that to `dolGetListOfObjectClasses` with a capital C, just to maintain consistency.

Reply:
yes, thank's